sdk/sqlmanagement/Azure.ResourceManager.Sql/CHANGELOG.md

```diff
@@ -4,6 +4,8 @@
 
 ### Features Added
```

Comment:
remove empty sections and update the `Unreleased` to 2025-05-14 then we can release it in quick.

Reply:
Added